event_sale_reservation/README.rst

```diff
@@ -0,0 +1,163 @@
+    :alt: OCA/event
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/event-12-0/event-12-0-event_sale_reservation
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/199/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module extends the functionality of event_sale to support selling
+reservations of events that still don't exist and to allow you to schedule the
+creation of events based on how many reservations already exist.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Install ``web_ir_actions_act_multi`` from https://github.com/OCA/web
+
+Configuration
+=============
+
+
+To make use of this module, a user needs these minimal permissions:
+
+- Sales / User: Own Documents Only
+- Events / User
+
+Usage
+=====
+
+To know how many reservations exist for a given event type:
+
+#. Go to *Events > Configuration > Event Categories* and pick or create one.
+#. There's a new smart button called *Reserved seats* with that count.
+#. Click on it to get to the sales orders where the seats got reserved.
+
+But that counter will be probably zero when you install, so let's see how to
+increase it.
+
+To create an event reservation product:
+
+#. Go to *Sales > Products > Products*.
+#. Create one.
+#. Set its basic info (name, price...) and go to *Sales* tab.
+#. Under *Events*, tick *Is an event reservation*.
+#. Select one *Event type for reservations*.
+#. Save.
+
+From now on, you can sell event reservations for that event type. To do it:
+
+#. Go to *Sales > Orders > Quotations*.
+#. Create one.
+#. Set its basic info (customer, date...) and go to *Order lines* tab.
+#. Click *Add a product*.
+#. Select the event reservation product you created above.
+#. Set its info (quantity, price...).
+#. Save that line and the quotation.
+
+At this point, the reservation is not yet confirmed, so if you go to the event
+type, the smart button will still count zero.
+
+To confirm those reservations:
+
+#. Go to the quotation you just created (if you are not there yet).
+#. Click on *Confirm*.
+
+Now, if you go to the event type form, the smart button will indicate how many
+reserved seats exist.
+
+If you want to convert those reservations into real event registrations:
+
+#. Go to the quotation you just created (if you are not there yet).
+#. Click on *Register in event*.
+#. In the wizard you see, set the *Event* and *Event Ticket* for all the order
+   lines you want to convert into registrations.
+#. If there is any line you still don't want to convert, remove it from the
+   wizard.
+#. Click on *Next*.
+#. A new wizard will appear, where you will be able to specify the name, email
```

Comment:
Doing the functional test I got stuck here, as the second wizard didn't pop up but the line got the event and ticket info but the registrations weren't made.

I guess has something to do with the `web_ir_actions_act_multi` action

Reply:
I pushed a fix, thanks. Could you try again?